tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -364,6 +364,7 @@ LogicalResult LowerHLOToLLVM(ModuleOp m, const DISCLoweringOptions& options) {
 
   auto& gpu_options = options.gpu_info;
 
+  pm.addNestedPass<FuncOp>(disc_ral::createDiscCommonRewriterPass());
```

Comment:
Not need to do such decomposition on cpu device

Reply:
done